release/models/local-routing/openconfig-local-routing.yang

```diff
@@ -298,11 +304,44 @@ module openconfig-local-routing {
           uses local-static-state;
         }
 
+        container next-network-instance {
```

Comment:
Should follow the precedence from the afts model where a lookup is intended to be triggered in a different, destination network-instance.  The leaf used there is `next-hop-group-network-instance`

[/network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/next-hop-group-network-instance](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-afts-ipv6-unicast-ipv6-entry-state-next-hop-group-network-instance)


A similar pattern for static-routes could be:

/network-instances/network-instance/protocols/protocol/static-routes/static/config/next-hop-group-network-instance

Reply:
Isn't this change analogous to https://github.com/openconfig/public/pull/1219, i.e., resolution of next-hop in a different network-instance?

If so, Isn't it better & flexible to emulate the same model here for IP routes as well? i.e., Adding a `next-hop-network-instance` leaf under `/network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/config`, same leaf should be added in `next-hop/state` container as well.

Please correct me if I completely misunderstood the intent of this PR.